.gitignore

```diff
@@ -131,7 +131,9 @@ dmypy.json
 
 # projest related
 #/fuseki
-/data-dev
+data-dev/
+fuseki-data/
```

Comment:
this folder gets created because of the change in this environment variable: `RDF_DIFFER_FUSEKI_DATA_FOLDER=fuseki-data`
we can probably remove `data-dev` from the ignore file